modules/util/get_set_value.js

```diff
@@ -32,11 +32,12 @@ export function utilGetSetValue(selection, value, shouldUpdate) {
         // see https://html.spec.whatwg.org/multipage/input.html#concept-input-apply
         const supportedTypes = ['text', 'search', 'url', 'tel', 'password'];
         return function() {
+            func.apply(this, arguments);
+
             if (!supportedTypes.includes(this.type)) {
                 return;
             }
             const cursor = { start: this.selectionStart, end: this.selectionEnd };
```

Comment:
Based on the code that changed, these regressions were probably introduced in 14c752548d2899b7411fcc8480ce3e2784df1253 (related to #8769 and 2b64d703523a2856bd41b894dff9fcd5dc4bdc75). I believe the intention was for this line to memoize the cursor position before setting the value in order to restore that position afterwards. It needs to go to the very top of this function before the call to `apply()`.